Source/Scene/Expression.js

```diff
@@ -261,7 +267,9 @@ define([
     }
 
     function setEvaluateFunction(node) {
-        if (node._type === ExpressionNodeType.BINARY) {
+        if (node._type === ExpressionNodeType.TERNARY) {
```

Comment:
Looking forward, from the AST perspective (not the tokenized perspective), perhaps `ExpressionNodeType.TERNARY` is better named `ExpressionNodeType.CONDITIONAL`.